controllers/ProposalController.php

```diff
@@ -647,26 +695,49 @@ private function checkIfCommentExists($commentId): Comment {
         return $comment;
     }
 
+    /**
+     * Check if User is owner of a comment.
+     * An admin can edit any comment.
+     *
+     * @param Comment $comment
+     */
     private function checkIfUserIsOwnerOfComment(Comment $comment) {
         $unauthorizedException = NotFoundHttpException::class;
 
+        if (MainController::getCurrentUser()->role == User::USER_ROLE_ADMIN) {
```

Comment:
Well thought out, however i think you should use '===' operator

Reply:
Deal